RELEASE.md

```diff
@@ -0,0 +1,4 @@
+Release type: patch
+
+This release allows conversion of pydantic models with mutable default fields into strawberry types.
+Also fixes bug when converting a pydantic model field with default_factory. Previously would except when it was declared before fields without defaults.
```

Comment:
I think there’s a typo here :)
```suggestion
Also fixes bug when converting a pydantic model field with default_factory. Previously it would raise an exception when fields with a default_factory were declared before fields without defaults.
```